src/hotspot/cpu/x86/macroAssembler_x86.cpp

```diff
@@ -26,6 +26,7 @@
 #include "jvm.h"
 #include "asm/assembler.hpp"
 #include "asm/assembler.inline.hpp"
+#include "c1/c1_FrameMap.hpp"
```

Comment:
Hi @tschatzl , this line breaks the build of VM with `--with-jvm-features=-compiler1`.
Please have a look.
Thanks.

Reply:
will do. Thanks!